bin/git-browse

```diff
@@ -2,7 +2,11 @@
 
 if [[ $1 == "" ]]
 then
-    remote=$(git remote | head -n 1)
+    remote=$(git remote | grep origin | head -n 1)
```

Comment:
Better to use `git config branch.$branch.remote || echo "origin"`, while the `$branch` is the current branch? So that we can get the remote of the current branch, or fallback to `origin`.

Reply:
Nice. I'll update the code.